src/emu/image.cpp

```diff
@@ -17,6 +19,13 @@
 #include "xmlfile.h"
 
 //**************************************************************************
+//  STATIC VARIABLES
+//**************************************************************************
+
+static std::regex s_softlist_regex("\\w+\\:\\w+\\:\\w+");
```

Comment:
wouldn't it be better to use a string like "[^:/]+\\:[^:/]+\\:[^:/]+" working from unacceptable characters rather than a narrow class to include?